src/main.cpp

```diff
@@ -240,6 +247,26 @@ int main(void)
                 firstFrame = true;
             }
         }
+
+        if(menuOption == INTERMEDIATE) {
```

Comment:
posibly name this screen == result. menuOption implies this is only for the menu not for the game. and intermediate is not very descriptive.